quickfixj-core/src/main/java/quickfix/mina/initiator/AbstractSocketInitiator.java

```diff
@@ -174,7 +177,7 @@ && getSettings().isSetting(sessionID, Initiator.SETTING_PROXY_DOMAIN)) {
         ScheduledExecutorService scheduledExecutorService = (scheduledReconnectExecutor != null ? scheduledReconnectExecutor : getScheduledExecutorService());
         try {
             final IoSessionInitiator ioSessionInitiator = new IoSessionInitiator(session,
-                    socketAddresses, localAddress, reconnectingIntervals,
+                    socketAddresses, localAddress, connectTimeout, reconnectingIntervals,
```

Comment:
After more thoughts on this, I think it would be a cleaner approach to move the SocketConnectTimeout to the `NetworkingOptions` class. That way we do not need to hand through yet another parameter. What do you think?

Reply:
yeah, sure
do you want me to make the change? I think you can edit PR as maintainer, this may be just quicker

Reply:
@AndreyNudko , quickly checked if this is possible. Turned out that `NetworkingOptions` only sets options in MINA's `SocketSessionConfig`, so moving the parameter to `NetworkingOptions` does not work.